qa/tasks/ceph_manager.py

```diff
@@ -1534,10 +1534,11 @@ def tmp(x):
         self.rook = rook
         self.cephadm = cephadm
         self.testdir = teuthology.get_testdir(self.ctx)
-        self.run_cluster_cmd_prefix = [
+        pre = [ # prefix args for ceph cmds to be executed
```

Comment:
It's better to move the comment out of `[]` ? Such as just a line up before this parameter ?

Reply:
Moved. Thanks!